wordpress-mobile/WordPress-iOS
# [Stats Refresh] Enable DWMY block loading feature in 13.7

This PR enables the block loading for everyone.

## To test:
- Make sure it builds
- Open Stats from a blog with data (en.blog)
- Open Insights and DWMY (details included)
- Everything should work as before

Update release notes:
